### ISSUE TITLE
document chrony setup/configuration

### ISSUE THREAD

**Issue by ipa-fxm** [ipa320/setup_cob4#118]
as done by @ipa-bnm on paul-ingolstadt to overcome buggy ntp time synchronization...
(first glance looked promising...long-term stress-test pending)

@ipa-bnm 
would you be so kind to drop a few words on what needs to be done and add example master and client configs from paul-ingolstadt?

@ipa-fmw 
if chrony proves to be the tool of choice, this needs to be adjusted in the auto-setup image/documentation #114 (or a fresh follow-up PR once the ntp-based setup is finished)

**Comment by ipa-bnm** [comment]
@ipa-fxm I'll put the chrony configs for master and slave and a documentation to `setup_cob4`

**Comment by ipa-fxm** [comment]
templates have been added in https://github.com/ipa320/setup_cob4/pull/129

**Comment by ipa-fxm** [comment]
@ipa-fmw @ipa-nhg @ipa-bnm
has this been done during your efforts regarding the setup from stick image? please reference PR and close then...

**Comment by ipa-fmw** [comment]
done in https://github.com/ipa320/setup_cob4/pull/131
